doc/lua_api.md

```diff
@@ -8802,6 +8802,10 @@ child will follow movement and rotation of that bone.
             but keeping original luma and being symmetrical in terms of saturation
             (eg. -1 and 1 is the same saturation and luma, but different hues)
         * This value has no effect on clients who have shaders or post-processing disabled.
+      * `foliage_translucency` controls the intensity of the foliage translucency effect (default: `1.5`).
+        * This has no effect when dynamic shadows are disabled and on clients who have the "Translucent foliage" effect disabled.
+      * `specular_intensity` controls the intensity of specular highlights on nodes and liquids (default: `1.5`).
```

Comment:
Same as above. What am I missing to see an effect? There is no difference when looking at water, or though water.